packages/mobile/src/account/Invite.tsx

```diff
@@ -85,10 +83,7 @@ class Invite extends React.Component<Props, State> {
 
     const hasGivenContactPermission = await requestContactsPermission()
     this.setState({ hasGivenContactPermission })
-
-    if (hasGivenContactPermission) {
```

Comment:
Why is checking contact permissions no longer necessary?

Reply:
We are already checking for permissions in the `doImportContacts` generator within `contactMapping.ts`. Beyond not needing the redundant check on the frontend, it's easier to only have one check on the backend so we only have to place the analytics event in one place.